geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/ClientReAuthenticateMessage.java

```diff
@@ -28,6 +28,11 @@
 public class ClientReAuthenticateMessage implements ClientMessage {
   @Immutable
   public static final KnownVersion RE_AUTHENTICATION_START_VERSION = KnownVersion.GEODE_1_15_0;
+
+  public static final String OLD_CLIENT_AUTHENTICATION_EXPIRED =
+      "Authorization expired for a client with a version less than Geode 1.15. Cannot re-authenticate an older client "
+          + " that has a server to client queue for CQs or interest registrations. "
```

Comment:
Instead of calling CQs or Interest; can we just say 
"older clients with client event queue"